orange/src/main/java/com/hartwig/hmftools/orange/algo/cuppa/CuppaDataFactory.java

```diff
@@ -100,4 +103,13 @@ static int getSvFeatureValue(@NotNull CuppaPredictions cuppaPredictions, @NotNul
 
         return (int) Math.round(predictionEntry.FeatureValue);
     }
+    static CuppaMode getCuppaMode(@NotNull CuppaPredictions cuppaPredictions) {
+        if (cuppaPredictions.ClassifierNames.contains(ClassifierName.COMBINED)) {
+            return CuppaMode.WGTS;
+        } else if (cuppaPredictions.ClassifierNames.contains(ClassifierName.DNA_COMBINED)) {
+            return CuppaMode.WGS;
+        } else {
```

Comment:
Looking at other enums in orange (eg. ChordStatus, PurpleDriverType, etc) it seems that the convention would be to use an UNKNOWN or perhaps NA enum value rather than null here. 

The ticket seems to imply that the other classifiers won't be used in Orange, is that the case?